Client/src/css/allSortsPlay.css

```diff
@@ -0,0 +1,79 @@
+.playContainer{
+    height: 90vh;
+    width: 100vw;
+    display: flex;
+    background-color: var(--white);
+    flex-direction: column;
+}
+
+.formContainer{
+    width: 100%;
+    height: 10%;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+
+}
+
+.visualize-container{
+    width: 94%;
+    padding-left: 6%;
+    height: 90%;
+    display: flex;
+    justify-content: center;
+    align-items: flex-end;
+}
+
+
+
+.sorting-label{
+    text-align:start;
+    font-weight: bolder;
+}
+
+.disabled{
+    color: gray;
+}
+
+.disabled:hover{
+    cursor: default;
+    color: grey;
+}
```

Comment:
If you're using grey, especially more than once, add it to the global variables

Reply:
Done